src/xdocs/config_coding.xml

```diff
@@ -3478,13 +3478,15 @@ public static class A {
   public A(int field1) {
     field1 = field1; // violation: Reference to instance variable "field1" needs "this".
     field2 = 0; // violation: Reference to instance variable "field2" needs "this".
+    String field2;
+    field2 = "0"; // No violation. Local var allowed
   }
 
   void foo3() {
     String field1 = "values";
     field1 = field1; // violation:  Reference to instance variable "field1" needs "this".
     }
```

Comment:
Since you fixed indentation of brace just below this one, this brace should be indented left 2 spaces.